src/context/user/UserContext.jsx

```diff
@@ -0,0 +1,16 @@
+import { createContext } from "react";
+import { useState,useContext } from "react";
+
+const UserContext = createContext();
+
+export const UserProvider = ({children}) => {
+
+  const [user,setUser] = useState({isLoggedIn: false});
```

Comment:
You might need `userName` or `userId` in future. So,  you can add the user to the state variable.

Reply:
After the user gets loggedin my `user` state will look like this.
and `_id` and `name` are accessible from it.
```json
{
  "_id": "15adeed5-070a-4ffd-828a-11c86a110d56",
  "firstName": "Adarsh",
  "lastName": "Balika",
  "email": "adarshbalika@gmail.com",
  "createdAt": "2022-03-19T11:29:18+05:30",
  "updatedAt": "2022-03-19T11:29:18+05:30",
  "cart": "[]",
  "wishlist": "[]",
  "id": "1",
  "isLoggedIn": true
}
```